src/d3a/d3a_core/sim_results/file_export_endpoints.py

```diff
@@ -170,10 +170,8 @@ def _specific_row(self, slot, market):
             desired = self.area.strategy.state.get_desired_energy_Wh(slot) / 1000
             return [desired, self._traded(market) + desired]
         elif isinstance(self.area.strategy, PVStrategy):
-            produced = self.area.strategy.state.available_energy_kWh.get(slot, 0)
-            return [produced,
-                    round(produced -
-                          self.area.strategy.energy_production_forecast_kWh.get(slot, 0), 4),
-                    self.area.strategy.energy_production_forecast_kWh.get(slot, 0)
-                    ]
+            produced = self.area.strategy.state.get_available_energy_kWh(slot, 0.0)
+            forecasted = self.area.strategy.state.get_energy_production_forecast_kWh(slot, 0.0)
+            curtailed = round(produced - forecasted, 4)
```

Comment:
minor, but how about renaming this to `residue`?

Reply:
Well, because specifically for the renewable power plants/devices, the operation of limiting the generating power is called curtailment, and the energy is called curtailed_energy. Residual energy is mostly encountered as a term for loads (and for a very specific circumstance, it refers to the remaining load in a grid after renewable devices have been introduced), and not for other devices. I know that it is not a big deal, but it will be misleading if someone looks at the code later. 

We use this term only in the trade object IIRC, in which IMO it is also not 100% correct. However, since I have not seen a commonly used term in energy markets for that, I am fine with keeping it there.

Reply:
Ah, ok. Thanks for explaining in detail. I am still getting used to the terms used in electrical grids and renewable power plants. So, of course: Keep it like it is.